lib/api/efd.js

```diff
@@ -0,0 +1,37 @@
+import { InfluxDB, flux } from "@influxdata/influxdb-client";
+
+export default function getEfd() {
+  const url = process.env.NEXT_PUBLIC_EFD_URL;
+  const bucket = process.env.NEXT_PUBLIC_EFD_BUCKET;
+  const token = process.env.NEXT_PUBLIC_EFD_TOKEN;
+
+  const influxDB = new InfluxDB({ url, token });
+  const queryApi = influxDB.getQueryApi("");
+
+  return new Promise((resolve, reject) => {
+    let data = {};
+    queryApi.queryRows(
+      flux`from(bucket: "${bucket}")
+      |> range(start: -60s)
+      |> filter(fn: (r) => r["_measurement"] == "lsst.sal.ESS.temperature" or r["_measurement"] == "lsst.sal.ESS.relativeHumidity" or r["_measurement"] == "lsst.sal.ESS.airFlow")
```

Comment:
trimmed down to just the current data that we need: temperature, humidity, wind, and need to get precipitation